File: js/DoublyLinkedList/Node.js
class Node {
    constructor(value, next, prev) {
        this.value = value;
        this.next = next;
        this.prev = prev;
    }

    getValue() {
        return this.value;
    }

    getNext() {
        return this.next;
    }

    getPrevious() {
        return this.prev;
    }
};

module.exports = Node;

File: java/src/rosetastone/linkedlist/Node.java
package rosetastone.linkedlist;

public class Node {

    public int value;
    public Node next;

    public Node(int value, Node next) {
        this.value = value;
        this.next = next;
    }

    protected void setNext(Node n) {
        this.next = n;
    }

    protected Node getNext() {
        return this.next;
    }

    protected int getValue() {
        return this.value;
    }
}
